babel.config.js

```diff
@@ -1,3 +1,20 @@
 module.exports = {
-  presets: ['module:metro-react-native-babel-preset']
+  presets: ['module:@react-native/babel-preset'],
+  plugins: [
+    [
+      'module-resolver',
+      {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+        root: ['.'],
+        alias: {
+          'react-native-calendars': './src/index.ts',
+          commons: './src/commons',
+          services: './src/services',
+          utils: './src/utils',
+          hooks: './src/hooks.ts',
+          style: './src/style.ts'
```

Comment:
Are you using these other aliases in src files? (commons, services, etc..) if not better remove them
If you do (I didn't find usages), we need to make sure the built version of Calendars go thru this babel, otherwise it might break consumers, including us